gnomad_qc/v4/annotations/vrs_annotation_batch.py

```diff
@@ -163,8 +166,17 @@ def main(args):
     ht_example.show()
 
     working_bucket = args.working_bucket
-
-    input_path = v4_input_ht().path
+    data_type = args.data_type
+    if args.input_path is not None:
+        input_path = args.input_path
```

Comment:
should raise a ValueError here if input_path is not None but output_vrs_path or output_vrs_anno_ori_path have not been specified